tree/dataframe/inc/ROOT/RDFActionHelpers.hxx

```diff
@@ -323,6 +324,69 @@ public:
    HIST &PartialUpdate(unsigned int slot) { return *fTo->GetAtSlotRaw(slot); }
 };
 
+class FillTGraphHelper : public ROOT::Detail::RDF::RActionImpl<FillTGraphHelper> {
+public:
+   using TGraph = ::TGraph;
+   using Result_t = TGraph;
+
+private:
+   std::shared_ptr<TGraph> fResultGraph;
+   std::unique_ptr<ROOT::TThreadedObject<TGraph>> fTo;
+   bool fIsSortingRequired;
+
+public:
+   FillTGraphHelper(FillTGraphHelper &&) = default;
+   FillTGraphHelper(const FillTGraphHelper &) = delete;
+
+   // The last parameter is always false, as at the moment there is no way to propagate the parameter from the user to this method
+   FillTGraphHelper(const std::shared_ptr<TGraph> &h, const unsigned int nSlots, const bool &isSortingRequired = false)
+      : fResultGraph(h), fTo(new ROOT::TThreadedObject<TGraph>(*h)), fIsSortingRequired(isSortingRequired)
+   {
+      fTo->SetAtSlot(nSlots, h);
```

Comment:
does it make sense to use as slot, say 0, the actual graph in order to save some memory? In this case the loop below should start at 1.